### ISSUE TITLE
Add support for actix-web 4

### ISSUE THREAD

**Issue by SorenHolstHansen** [lambda-fairy/maud#348]
As the title says. It currently breaks on an actix web route with
```
the trait bound `<index as actix_web::service::HttpServiceFactory>::register::index::{closure#0}::maud::PreEscaped<std::string::String>: actix_web::Responder` is not satisfied
the following other types implement trait `actix_web::Responder`:
  &'static [u8]
  &'static str
  &std::string::String
  (R, actix_web::http::StatusCode)
  actix_http::responses::builder::ResponseBuilder
  actix_web::CustomizeResponder<T>
  actix_web::Either<L, R>
  actix_web::HttpResponse<B>
and 13 others
required because of the requirements on the impl of `actix_web::Responder` for `std::result::Result<<index as actix_web::service::HttpServiceFactory>::register::index::{closure#0}::maud::PreEscaped<std::string::String>, actix_web::Error>
```

**Comment by SanchithHegde** [comment]
Related: #330, #340.

TL;DR: Use `maud` as a [git dependency](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories).

**Comment by SorenHolstHansen** [comment]
Thanks
